Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_end/translation.txt

```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=
+「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone ...... I've fallen in love with it!”
```

Comment:
```suggestion
「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone... I've fallen in love with it!”
```

maybe make the second bit sound less passive?

```suggestion
「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone... I love it!”
```

Reply:
I went with fallen in love to keep consistency with some of the other uses of meromero both from KK's kouhai character and one of the other usages in a different conversation. I like that it has a slightly different nuance than just saying she loves it.